site/en/blog/chrome-ux-report-history-api/index.md

```diff
@@ -127,23 +125,25 @@ The overall shape of a response is similar - but there is a lot more data! There
+In this example, the `densities` time series for the 0 to 2500 ms bucket of the [Largest Contentful Paint (LCP)](https://web.dev/lcp/) metric is [`0.9190, 0.9203, 0.9194, 0.9195, 0.9183, 0.9187].` Each of these densities was observed during the corresponding `collectionPeriods` entry. For example, the fifth density, 0.9183, was the density for the fifth collection period, ending in September 3rd, 2022, and 0.9187 was the density in the period ending the week after that.
 
-In other words, interpreting the last time series entries in the example, for [https://web.dev](https://web.dev), we found that from August 14, 2022 until September 10, 2022, 91.87% of page loads had LCP values smaller than 2500ms, 5.27% had values between 2500ms and 4000ms, and 2.85% had values greater than 4000ms.
+In other words, interpreting the last time series entries in the example for [https://web.dev](https://web.dev), it was found that from August 14, 2022 until September 10, 2022, 91.87% of page loads had LCP values smaller than 2500ms, 5.27% had values between 2500ms and 4000ms, and 2.85% had values greater than 4000ms.
 
-Similarly, there’s a time series for the p75 values: the LCP p75 for August 14, 2022 until September 10, 2022 was 1377. This means that for this collection period, 75% of user experiences had an LCP of less than 1377 ms, and 25% of user experiences had an LCP greater than 1377 ms.
+Similarly, there’s a time series for the p75 values: the LCP p75 for August 14, 2022 until September 10, 2022 was 1377. This means that, for this collection period, 75% of user experiences had an LCP of less than 1377 ms, and 25% of user experiences had an LCP greater than 1377 ms.
 
 While the example only lists 6 time series entries and collection periods, responses from the API currently provide 25 time series entries; since the end dates for each of these collection periods are Saturdays that are 7 days apart, this covers 6 months.
 
 {% Aside %}
-Unlike the daily API, the CrUX History API is only available once a week. However, like the daily API, each value in the CrUX History API is measured from the previous 28 days. This means the CrUX History API should represent the same data that was available via the daily API for the given date—it is just now available to query retrospectively. However, it does mean the CrUX History API contains overlapping periods of data but, as a trend analysis, it is useful to track the change in metrics over time.
+
+Unlike the daily API, the historical API is only available once a week. However, like the daily API, each value in the historical API is measured from the previous 28 days. This means the historical API should represent the same data that was available via the daily API for the given date—it is just now available to query retrospectively. However, it does mean the historical API contains overlapping periods of data but—as a trend analysis—it is useful to track the change in metrics over time.
```

Comment:
It looks like all the "CrUX History API" mentions reverted back to "historical API". I think we had this standardized to "CrUX History API" earlier and maybe that's nicer for consistency? Maybe CTRL+F across the file to fix these up?

Reply:
Fixed.